### PR TITLE
Add Log Statements

### PR DESCRIPTION
**Pull request by tymefighter**

### CODE REVIEW

**Comment by tymefighter** [comment]
@iamrakesh28 @sakshi1603 Can you take a look at this one

**Comment by sakshi1603** [comment]
Please update the logs.

**Review by iamrakesh28 — APPROVED** [review]

**Comment by sakshi1603** [comment]
You missed to update few logs. Please update them.

File: src/processing/shape/LineDrawer.java [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 import java.util.ArrayList;
 import processing.utility.*;
+import infrastructure.validation.logger.LoggerFactory;
+import infrastructure.validation.logger.ILogger;
+import infrastructure.validation.logger.LogLevel;
+import infrastructure.validation.logger.ModuleID;
```
Comment (iamrakesh28) [inline src/processing/shape/LineDrawer.java:8]: Add reviewer's name (my name) in this file.

**Review by iamrakesh28 — CHANGES_REQUESTED** [review]

File: src/processing/shape/LineDrawer.java (hunk 1 above)
Comment (tymefighter) [reply src/processing/shape/LineDrawer.java:8]: ok I'll add now

**Comment by tymefighter** [comment]
@sakshi1603 @iamrakesh28 I have done the changes, we can merge now I think !!

File: src/processing/boardobject/BoardObject.java [hunk 2]
```diff
@@ -31,123 +35,197 @@
...
+     * @param isReset Boolean representing whether object is a reset object or not
+     */
+    public BoardObject (
+        ArrayList <Pixel> pixels,
+        ObjectId objectId,
+        Timestamp timestamp,
+        UserId userId,
+        boolean isReset
+    ) {
+    	LoggerFactory.getLoggerInstance().log(
```
Comment (iamrakesh28) [inline src/processing/boardobject/BoardObject.java:76]: Thread id should be added in the log

File: src/processing/boardobject/BoardObject.java [hunk 3]
```diff
@@ -31,123 +35,197 @@
...
+    	this.boardOp = new CreateOperation();
+    	this.objectId = objectId;
+    	this.timestamp = timestamp;
+    	this.isReset = isReset;
+    	this.prevPixelIntensities = null;
+    }
+    
+    /** Gets the operation corresponding to this shape */
+    public IBoardObjectOperation getOperation() {
+    	LoggerFactory.getLoggerInstance().log(
```
Comment (iamrakesh28) [inline src/processing/boardobject/BoardObject.java:93]: Thread id should be added in the lg

File: src/processing/boardobject/BoardObject.java [hunk 4]
```diff
@@ -31,123 +35,197 @@
...
+		);
+    	return boardOp;
+    }
+    
+    /**
+     * Sets the operations to be performed on this board object
+     * 
+     * @param boardOp The operation to be performed on this board object
+     */
+    public void setOperation (IBoardObjectOperation boardOp) {
```
Comment (iamrakesh28) [inline src/processing/boardobject/BoardObject.java:107]: Thread id

File: src/processing/boardobject/BoardObject.java [hunk 5]
```diff
@@ -31,123 +35,197 @@
...
+    	LoggerFactory.getLoggerInstance().log(
+			ModuleID.PROCESSING, 
+			LogLevel.INFO, 
+			"Set Operation of the Board Object"
+		);
+    	this.boardOp = boardOp;    		
+    }
+
+    /** Gets the board object's list of pixels */
+    public ArrayList <Pixel> getPixels () {
```
Comment (iamrakesh28) [inline src/processing/boardobject/BoardObject.java:118]: Thread id

File: src/processing/boardobject/BoardObject.java [hunk 6]
```diff
@@ -31,123 +35,197 @@
...
+			ModuleID.PROCESSING, 
+			LogLevel.INFO, 
+			"Get Pixels from the Board Object"
+		);
+    	return pixels;
+    }
+    
+    /** Gets the board object's list of positions */
+    public ArrayList <Position> getPositions () {
+    	LoggerFactory.getLoggerInstance().log(
```
Comment (iamrakesh28) [inline src/processing/boardobject/BoardObject.java:130]: Thread id

File: src/processing/boardobject/BoardObject.java [hunk 7]
```diff
@@ -31,123 +35,197 @@
...
+    	// the array list of positions
+    	for(Pixel p : pixels)
+    		positions.add(p.position);
+    	
+    	return positions;
+    }
+
+    /** Sets the pixels using the given array-list of pixels */
+    public void setPixels (ArrayList <Pixel> pixels) {
+    	LoggerFactory.getLoggerInstance().log(
```
Comment (iamrakesh28) [inline src/processing/boardobject/BoardObject.java:150]: Thread id missing

File: src/processing/shape/BoardObjectBuilder.java [hunk 8]
```diff
@@ -34,17 +38,35 @@ public static BoardObject drawCircle (
         Radius radius,
         Intensity intensity
     ) {
+    	ILogger logger = LoggerFactory.getLoggerInstance();
+    	
+    	logger.log(ModuleID.PROCESSING, LogLevel.INFO, "Drawing The Circle");
```
Comment (iamrakesh28) [inline src/processing/shape/BoardObjectBuilder.java]: Thread id

File: src/processing/shape/BoardObjectBuilder.java [hunk 9]
```diff
@@ -64,16 +86,34 @@ public static BoardObject drawCircleFill (
         Radius radius,
         Intensity intensity
     ) {
+    	ILogger logger = LoggerFactory.getLoggerInstance();
+    	
+    	logger.log(ModuleID.PROCESSING, LogLevel.INFO, "Drawing The Filled Circle");
```
Comment (iamrakesh28) [inline src/processing/shape/BoardObjectBuilder.java]: Thread id

**Review by iamrakesh28 — CHANGES_REQUESTED** [review]
Put thread id in the logs

File: src/processing/shape/CircleDrawer.java [hunk 10]
```diff
@@ -93,6 +105,12 @@ public static void setAlgorithmFill(AlgorithmFill algorithm) {
 			// If Devansh method was selected, use Devansh Circle
 			// Filling Algorithm
```
Comment (sakshi1603) [inline src/processing/shape/CircleDrawer.java]: please change multiple line comment format to /* */

File: src/processing/shape/CircleDrawer.java [hunk 11]
```diff
@@ -103,6 +121,12 @@ public static void setAlgorithmFill(AlgorithmFill algorithm) {
 			// By Default, use Mid Point
 			// Based Filling Algorithm
```
Comment (sakshi1603) [inline src/processing/shape/CircleDrawer.java]: Please change here also

**Comment by sakshi1603** [comment]
Please correct multiple line comment format in CircleDrawer.java, LineDrawer.java, RectangleDrawer.java and ShapeHelper.java

**Pull request merged**